nodelet/src/nodelet.cpp

```diff
@@ -89,7 +89,7 @@ class NodeletArgumentParsing
     bool is_bond_enabled_;
 
   public:
-    //NodeletArgumentParsing() { };
+    NodeletArgumentParsing() : is_bond_enabled_(true) { };
```

Comment:
Seems like you can also remove the `is_bond_enabled_ = true;` at the beginning of `parseArgs`.
Otherwise it looks good to me.

Reply:
Thanks for the suggestion, I removed it :+1:.